### DISCUSSION TITLE
dashed list style types & underlined links

### DISCUSSION THREAD

**Discussion by diomed** [tailwindlabs/tailwindcss#11516] [Ideas]
please give us **dashed** list style types and links that underline from left to right on hover by some kind of **default** classes

**Comment by mybearworld** [comment]
Dashed list style types are not part of CSS so Tailwind can't have a utility for that, but you can use a string, with `list-['-_']`.
Links that have underlines on hover can be done with the easy to understand `hover:underline`.
[Play](https://play.tailwindcss.com/j21jjE04le)

**Comment by diomed** [comment]
> Dashed list style types are not part of CSS so Tailwind can't have a utility for that, but you can use a string, with `list-['-_']`. Links that have underlines on hover can be done with the easy to understand `hover:underline`. [Play](https://play.tailwindcss.com/j21jjE04le)

```
  <ul class="list-inside list-['-_']">
```
that's a very nice solution, thank you.

As far as links go, I know about `hover:underline`. 

I asked about hover style as in first example [here](https://tympanus.net/Development/LineHoverStyles) which makes them go from left to right
